### PR TITLE
✨ Invibes ads: new ad parameter

### PR DESCRIPTION
**Pull request by invibes**
- Add a new parameter that will be used in our testing environment.

### CODE REVIEW

**Comment by invibes** [comment]
@googlebot I consent.

**Comment by aurelvasile** [comment]
@googlebot I consent.

**Comment by rcheptanariu** [comment]
@googlebot I consent.

**Comment by aurelvasile** [comment]
@googlebot I fixed it.

File: examples/ads.amp.html [hunk 1]
```diff
@@ -1352,12 +1352,13 @@ <h2>Insticator</h2>
       data-site-id="c40f316f-e18a-4cef-b9a6-b04acbaec113"
       data-embed-id="77ee0296-8ac4-4bca-91c8-51256b9830f1">
   </amp-embed>
-
+
```
Comment (powerivq) [inline examples/ads.amp.html:1355]: Nit: remove space

**Pull request merged**